### ISSUE TITLE
Crash to Desktop on Join(Update 120)

### ISSUE THREAD

**Issue by AWPgodkiller** [polarathene/biglobby#29]
Sorry for my English.
I've experienced a crash when i join friend's game or they join mine.
Both my friends and i have installed the latest version of BigLobby(v2.34) and have install the .pdmod file with Bundle Modder(v1.16.3.1).

Here is the crash report:

Crash date: 31/10/2016 19:19:58

PAYDAY 2 Crash log
Application has crashed: C++ exception
[string "lib/managers/menu/menulobbyrenderer.lua"]:210: attempt to index local 'slot' (a nil value)
SCRIPT STACK
-------------------------------
Callstack:
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
-------------------------------
Current thread: Main
-------------------------------
System information:
	Application version : 1.56.66
	CPU : Intel(R) Core(TM) i5-3330 CPU @ 3.00GHz (2 cores); SSE; SSE2; SSE3; SSSE3; SSE4.1; SSE4.2
	DirectX : 10.0 
	GPU : NVIDIA GeForce GTX 970 / nvd3dum.dll[21.21.13.7290]
	Language : @ID94cca2fdaccfc2cd@
	Memory :     16330MB 160KB
	OS : 6.1.7601 (Service Pack 1) 0x100-0x1 (64 bits)
	Physics : threaded
	Renderer : DX9 threaded
	Sound : Microsoft (喇叭 (2- USB PnP Sound Device))

I am looking forward for your reply, thanks!

PS:To be exact, the crash happened when my friends and i enter the Briefing menu ( I mean the menu         that you can buy doctor bag or ammo bag)

**Comment by polarathene** [comment]
Is one of you using game controller instead of keyboard?

Try use this file from [v2.32](https://github.com/polarathene/biglobby/releases/tag/v2.32): `lua/lib/managers/menu/_missionbriefinggui.lua`, let me know if fix problem.
 If not it might be this one: `lua/lib/managers/menu/_menulobbyrenderer.lua`.

If still crash, these are other files that change:

```
lua/lib/managers/hud/_hudlootscreen.lua
lua/lib/managers/hud/_hudmissionbriefing.lua
lua/lib/managers/hud/_hudteammate.lua
lua/lib/managers/menu/_contractboxgui.lua
lua/lib/managers/menu/_menucomponentmanager.lua
```

**Comment by AWPgodkiller** [comment]
Thx for quick reply, i will try it!

**Comment by polarathene** [comment]
@AWPgodkiller did it work?

**Comment by AWPgodkiller** [comment]
Sorry for my VERY late reply. But the crash bug seem been fixed when we use the v2.32  menulobbyrenderer.lua . Thank you very much!!
